README.md

```diff
@@ -543,15 +540,15 @@ spec:
 
 ## Reconciler Implementation
 
-A reconciler implementation **MUST** support mapping to PodSpec-able resources without defining a Application Resource Mapping for those types.  If no Application Resource Mapping exists for the `ServiceBinding` application resource type and the application resource is not PodSpec-able, the reconciliation **MUST** fail.
+A reconciler implementation **MUST** support mapping to PodSpec-able resources without defining a Workload Resource Mapping for those types.  If no Workload Resource Mapping exists for the `ServiceBinding` workload resource type and the workload resource is not PodSpec-able, the reconciliation **MUST** fail.
```

Comment:
I think we should do a global search replace of Workload Resource Mapping for `WorkloadResourceMapping`, since we refer to the `kind` when talking about other CRs (like `ServiceBinding`)